### ISSUE TITLE
Routine bridge updates causes tests to fail with missing "resource_name"

### ISSUE THREAD

**Issue by t0yv0** [pulumi/pulumi-confluentcloud#248]
### What happened?

https://github.com/pulumi/pulumi-confluentcloud/pull/247

Need to investigate if this is a regression.

### Example

N/A

### Output of `pulumi about`

N/A

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by mikhailshilkov** [comment]
@t0yv0 Are you looking into this or should I assign @iwahbe (ops)?

**Comment by t0yv0** [comment]
Appreciate ops help.

**Comment by iwahbe** [comment]
The test was always broken, the bridge update caused it to be enforced. The relevant call was:

https://github.com/pulumi/pulumi-confluentcloud/blob/93707629eca3decc95713467800c1d4338947cc6/examples/basic-kafka-acls/py/__main__.py#L95

This was a typo:
```patch
-                              resource_name_=resource_name,
+                              resource_name=resource_name,
```

**Comment by t0yv0** [comment]
Thanks for looking into this. Good news!

**Comment by justinvp** [comment]
> This was a typo:

Actually, it wasn't a typo. More details here: https://github.com/pulumi/pulumi-confluentcloud/pull/247#issuecomment-1783717144

The example should go back to:

```
resource_name_=resource_name,
```

And the issue will be addressed after upgrading codegen to v3.91.1 of Pulumi, which reverts the change that introduced `_configure` in the first place.

**Comment by iwahbe** [comment]
I assumed it was a typo since `_configure` was checking for `"resource_name"` and not `"resource_name_"`. We have upgraded to v3.91.1 as part of https://github.com/pulumi/pulumi-confluentcloud/pull/255.
